### DISCUSSION TITLE
Glad to see Pinax as alive and kicking

### DISCUSSION THREAD

**Discussion by lmaloney** [pinax/pinax#169] [General]
It's been a while since I started a fresh project with Pinax.  I've spent all day looking for something similar to Pinax, and frankly I haven't found what I've wanted... all of which Pinax provides.

I was worried that it was dead, but it looks like there are updates being done.

**Comment by xyzPeter** [comment]
I'm new here, but really interested to explore and contribute to this project if it is being maintained.

Is anyone actively maintaining this project? 

Cheers,
Peter
